### PR TITLE
Fixes rich results

### PR DESCRIPTION
**Pull request by tlgimenes**
Brand was missing from product query. This made rich results to fail. 

This PR adds it back, thus fixing rich results

[marinbrasil](https://github.com/vtex-sites/marinbrasil.store/pull/116)

### CODE REVIEW

File: packages/gatsby-theme-vtex/src/components/ProductPage/SEO/index.tsx [hunk 1]
```diff
@@ -13,22 +13,19 @@ const SEO: FC<ProductPageProps> = (props) => {
...
+      {structuredProduct != null ? (
+        <Helmet
+          script={[
+            {
+              type: 'application/ld+json',
+              innerHTML: JSON.stringify(structuredProduct),
+            },
+          ]}
+        />
+      ) : null}
```
Comment (lariciamota) [inline packages/gatsby-theme-vtex/src/components/ProductPage/SEO/index.tsx:28]: Is there some reason for not using:
```suggestion
      {structuredProduct != null && (
        <Helmet
          script={[
            {
              type: 'application/ld+json',
              innerHTML: JSON.stringify(structuredProduct),
            },
          ]}
        />
      )}
```
?

Comment (tlgimenes) [reply packages/gatsby-theme-vtex/src/components/ProductPage/SEO/index.tsx:28]: `structuredProduct != null` is false for `undefined`. This means we may end up rendering `undefined`, which is, as far as I know, not allowed by React

Comment (lariciamota) [reply packages/gatsby-theme-vtex/src/components/ProductPage/SEO/index.tsx:28]: 🤔 I've used it a few times... [here](https://pt-br.reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) it says React ignores if it returns false and doesn't render anything

**Review by lucis — APPROVED** [review]

File: packages/gatsby-theme-vtex/src/components/ProductPage/SEO/index.tsx (hunk 1 above)
Comment (tlgimenes) [reply packages/gatsby-theme-vtex/src/components/ProductPage/SEO/index.tsx:31]: Cool !! I didn't know about. I'm changing it now

**Pull request merged**